### ISSUE TITLE
Tree: Accessibility role="treeitem"

### ISSUE THREAD

**Issue by kyybo** [primefaces/primereact#4692]
### Describe the bug

Hello,
In Tree componant, the attribute 'role="treeitem"' should be positioned on the 'li' tags and not on the 'div' tags .

### PrimeReact version

9.6.0

### React version

18.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Steps to reproduce the behavior

1. Create a basic Tree
`<Tree value={nodes} className="w-full md:w-30rem" />`

2. Check DOM :
```
<ul class="p-tree-container" role="tree" data-pc-section="container">
  <li class="p-treenode" data-pc-section="node">
    <div class="p-treenode-content" tabindex="0" role="treeitem" aria-posinset="1" aria-selected="false" data-pc-section="content">...</div>
  </li>
</ul>
```

### Expected behavior

Actual :
```
<ul class="p-tree-container" role="tree" data-pc-section="container">
  <li class="p-treenode" data-pc-section="node">
    <div class="p-treenode-content" tabindex="0" role="treeitem" aria-posinset="1" aria-selected="false" data-pc-section="content">...</div>
  </li>
</ul>
```

Expected :
```
<ul class="p-tree-container" role="tree" data-pc-section="container">
  <li class="p-treenode" data-pc-section="node" role="treeitem">
    <div class="p-treenode-content" tabindex="0" aria-posinset="1" aria-selected="false" data-pc-section="content">...</div>
  </li>
</ul>
```

**Comment by melloware** [comment]
Accessibility across the board is going to be addressed in a next major version and PrimeVue has already received a lot of these updates.  PrimeReact will receive them next!  I did make this fix though as its obvious.

**Comment by kyybo** [comment]
@melloware Thank you for your quick responses.
Is it worth reporting other problems like this if I find them, or should I wait for the next major version?

**Comment by melloware** [comment]
I think you will find a lot of them what I would say is look at PrimeVue showcase and if it has an issue then yes report it, if not i would say wait as anything you see in PrimeVue is getting rolled into here.

See: https://primevue.org/installation/
